libkbfs/block_retrieval_worker_test.go

```diff
@@ -85,6 +86,12 @@ func (bg *fakeBlockGetter) getBlock(ctx context.Context, kmd KeyMetadata, blockP
 	}
 }
 
+func (bg *fakeBlockGetter) assembleBlock(ctx context.Context,
```

Comment:
You could save some characters by using `_` as the param names, but whatever you prefer is fine with me.